components/SeniorSignup.tsx

```diff
@@ -0,0 +1,115 @@
+import React, {useState} from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  TouchableWithoutFeedback,
+  Keyboard,
+} from 'react-native';
+import {TextInput} from 'react-native-gesture-handler';
+
+const SeniorSignup = () => {
+  // inital state for the forms
+  const initialState = {
+    firstName: '',
+    lastName: '',
+    phoneNumber: '',
+    zipCode: '',
+    password: '',
+  };
+
+  // state for the forms
+  const [form, setForm] = useState(initialState);
+  // console.log('state change', form);
+
+  // handle submit when submit button is clicked
+  const handleSubmit = () => {};
+
+  return (
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss} accessible={false}>
+      <View style={styles.container}>
+        <Text>Sign Up</Text>
+        <Text style={styles.labels}>First Name</Text>
+        <TextInput
+          style={styles.inputFields}
+          placeholder="First Name"
+          onChangeText={text => setForm({...form, firstName: text})}
+          value={form.firstName}
+        />
+        <Text style={styles.labels}>Last Name</Text>
+        <TextInput
+          style={styles.inputFields}
+          placeholder="Last Name"
+          onChangeText={text => setForm({...form, firstName: text})}
+          value={form.firstName}
+        />
+        <Text style={styles.labels}>Phone Number</Text>
+        <TextInput
+          style={styles.inputFields}
+          keyboardType={'phone-pad'}
+          placeholder="1234567890"
+          onChangeText={text => setForm({...form, phoneNumber: text})}
+          value={form.phoneNumber}
+        />
+        <Text style={styles.labels}>Zip Code</Text>
+        <TextInput
+          style={styles.inputFields}
+          placeholder="Zip Code"
+          keyboardType={'numeric'}
+          onChangeText={text => setForm({...form, zipCode: text})}
+          value={form.zipCode}
+        />
+        <Text style={styles.labels}>Password</Text>
+        <TextInput
+          style={styles.inputFields}
+          secureTextEntry={true}
+          placeholder="Password"
+          onChangeText={text => setForm({...form, password: text})}
+          value={form.password}
+        />
```

Comment:
should we have a password confirmation field? older folks = more prone to typos.